app/views/api/alerts/index.html.slim

```diff
@@ -0,0 +1,59 @@
+- content_for :page_header_title, presenter.page_title
+
+- if presenter.empty_state?
+  tr role="row"
```

Comment:
It would probably be nice to make `app/views/shared/_empty_search_state.html.slim` more generic and cover both scenarios: empty state and empty search.

Reply:
Yes, I think I have a JIRA for that. I would like to do it for all the existing empty views without messing with this PR.